hi-ml-histopathology/src/histopathology/scripts/generate_crossval_report.py

```diff
@@ -115,12 +197,46 @@ def render_roc_and_pr_curves(report: HTMLReport, heading: str, level: int,
     report.add_images([roc_pr_curves_fig_path], base64_encode=True)
 
 
+def render_confusion_matrices(report: HTMLReport, heading: str, level: int, class_names: List[str],
+                              parent_run_id: str, aml_workspace: Workspace, report_dir: Path,
+                              output_filename: str, overwrite: bool, prefix: str = '') -> None:
+    """
+    Function to render confusion matrices for HTML reports.
+
+    :param report: HTML report to perform the rendering.
+    :param heading: Heading of the section.
+    :param level: Level of the section (e.g. sub-section, sub-sub-section, and so on).
```

Comment:
How does this work? Is there any documentation mapping numbers to levels?

Reply:
This corresponds to [HTML heading levels](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements), as described in the docs for `HTMLReport.add_heading()`.

Reply:
This corresponds to [HTML heading levels](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements), as described in the docs for `HTMLReport.add_heading()`.

Reply:
Thanks both, I have now modified the description of `level`